pkg/console/subresource/consoleserver/types.go

```diff
@@ -44,6 +46,20 @@ type ClusterInfo struct {
 	MasterPublicURL    string `yaml:"masterPublicURL,omitempty"`
 }
 
+// Monitoring holds URLs for monitoring related services
+type MonitoringInfo struct {
+	AlertmanagerURL string `yaml:"alertmanagerURL,omitempty"`
+	GrafanaURL      string `yaml:"grafanaURL,omitempty"`
+	PrometheusURL   string `yaml:"prometheusURL,omitempty"`
+	ThanosURL       string `yaml:"thanosURL,omitempty"`
+}
+
+// Logging holds URLs for logging related services
+type LoggingInfo struct {
+	KibanaAppURL      string `yaml:"kibanaAppURL,omitempty"`
+	KibanaInfraAppURL string `yaml:"kibanaInfraAppURL,omitempty"`
```

Comment:
This one is not used(so far) but cant hurt to propagate it to the server

Reply:
👍